Mergin/repair.py

```diff
@@ -0,0 +1,23 @@
+from qgis.core import QgsProject
+
+from .utils import project_grids_directory, copy_datum_shift_grids
+
+
+def fix_datum_shift_grids(mp):
+    """
+    Copies datum shift grids to the MerginMaps project directory.
+    Returns None on success and error message if some grids were not
+    copied.
+    """
+    if mp is None:
+        return "Invalid MerginMaps project"
```

Comment:
I think the right way of spelling MM is two separate words:
```suggestion
        return "Invalid Mergin Maps project"
```